packages/core/src/metrics/exports-default.ts

```diff
@@ -38,9 +39,20 @@ function gauge(name: string, value: number, data?: MetricData): void {
   metricsCore.gauge(MetricsAggregator, name, value, data);
 }
 
+/**
+ * Returns the metrics aggregator for a given client.
+ */
+function getMetricsAggregatorForClient(client: Client): MetricsAggregatorInterface {
+  return metricsCore.getMetricsAggregatorForClient(client, MetricsAggregator);
+}
+
 export const metricsDefault = {
   increment,
   distribution,
   set,
   gauge,
+  /**
+   * @ignore This is for internal use only.
+   */
+  getMetricsAggregatorForClient,
```

Comment:
Actually should we export this as a separate method to help with treeshaking?

Reply:
The `getMetricsAggregatorForClient` function is in the call chain for all the other `Sentry.metrics.*` functions so there will be no impact to tree shaking.